### ISSUE TITLE
"Ask Copilot" chat feature either in Xcode or in the menubar icon

### ISSUE THREAD

**Issue by cjgaspari** [github/CopilotForXcode#8]
Really excited for Github Copilot for Xcode. It would be great if users had feature parity with Visual Studio Code, specifically the ability to chat with Copilot for "rubber-duck" sessions. Code completion is great, but sometimes using chat helps to understand things more comprehensively. 

While there may be limitations within Xcode to enable a chat interface, I think a nice alternative would be in the menubar where the Copilot icon appears.

**Comment by khayashida919** [comment]
need chat and inlinee question

**Comment by HimashaHerath** [comment]
yeah I've been using this extension in the past few days works great and the extention is quite helpful in my work. If a chat feature can be added it would be awesome. Nice work on the extention

**Comment by testforstephen** [comment]
Hi all, we released Chat support in the latest Xcode copilot version for public preview. Please have a try.

See the doc on how to use it. https://github.com/github/copilotforxcode
